civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -1625,6 +1625,13 @@ function civihr_employee_portal_menu() {
         'type' => MENU_CALLBACK,
     );
 
+    $items['manager_approval/%ctools_js/quick_approval'] = array(
```

Comment:
maybe I Am missing something here but rout here is 

``php
manager_approval/%ctools_js/quick_approval
```

but in the js file it is :

```php 
manager_approval/ajax/quick_approval/
```